### ISSUE TITLE
Coupon with minimum amount doesn't apply in manual order

### ISSUE THREAD

**Issue by plxrisa** [woocommerce/woocommerce#22245]
**Describe the bug**
Coupon with minimum amount doesn't work correctly when you try to apply it in a manual order.

**To Reproduce**
1. Set US as your country.
2. Set that you selling only to US.
3. Set a tax rate 24%.
4. Create a product with price 15.50$
5. Create a coupon (percentage or fixed), with minimum spend 15.00$
6. Add the product to your cart and try to redeem the coupon.
7. You will see that works correctly.
8. Now try to add an order manually.
9. Add the product and click to recalculate.
10. Try to apply the coupon.
11. You will this error message "The minimum spend for this coupon is 15.00$"

Extra Info:
My settings in tax tab are:
1. Yes, I will enter prices inclusive of tax
2. Shop base
3. Standard
4. Rounding (unchecked)
5. Display prices shop/cart/checkout ---> Including
6. Display tax totals as a single total

**Screenshots**
You can check here for more details:
https://www.useloom.com/share/f9d7e98d595841f79a7c4b47c1d4a3bb

**Expected behavior**
You must be able to apply the coupon to the order manually, if the order TOTAL is above 15.00$.

**Isolating the problem (mark completed items with an [x]):**
- [x ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x ] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
`
### WordPress Environment ###

Home URL: http://localhost/woo
Site URL: http://localhost/woo
WC Version: 3.5.2
Log Directory Writable: ✔
WP Version: ❌ 4.9.9 - There is a newer version of WordPress available (5.0.1)
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: el
External object cache: –

### Server Environment ###

Server Info: Apache/2.4.35 (Win64) PHP/7.2.10
PHP Version: 7.2.10
PHP Post Max Size: 8 MB
PHP Time Limit: 120
PHP Max Input Vars: 2500
cURL Version: 7.59.0
OpenSSL/1.1.0i

SUHOSIN Installed: –
MySQL Version: 5.7.23
Max Upload Size: 2 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.5.2
WC Database Prefix: wp_
Total Database Size: 0.63MB
Database Data Size: 0.50MB
Database Index Size: 0.13MB
wp_woocommerce_sessions: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_api_keys: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_attribute_taxonomies: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_downloadable_product_permissions: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_order_items: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_order_itemmeta: Data: 0.00MB + Index: 0.01MB
wp_woocommerce_tax_rates: Data: 0.00MB + Index: 0.01MB
wp_woocommerce_tax_rate_locations: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_shipping_zones: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_shipping_zone_locations: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_shipping_zone_methods: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_payment_tokens: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_payment_tokenmeta: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_log: Data: 0.00MB + Index: 0.00MB
wp_commentmeta: Data: 0.00MB + Index: 0.00MB
wp_comments: Data: 0.00MB + Index: 0.01MB
wp_links: Data: 0.00MB + Index: 0.00MB
wp_options: Data: 0.47MB + Index: 0.03MB
wp_postmeta: Data: 0.02MB + Index: 0.02MB
wp_posts: Data: 0.01MB + Index: 0.01MB
wp_termmeta: Data: 0.00MB + Index: 0.01MB
wp_terms: Data: 0.00MB + Index: 0.01MB
wp_term_relationships: Data: 0.00MB + Index: 0.00MB
wp_term_taxonomy: Data: 0.00MB + Index: 0.00MB
wp_usermeta: Data: 0.00MB + Index: 0.01MB
wp_users: Data: 0.00MB + Index: 0.01MB
wp_wc_download_log: Data: 0.00MB + Index: 0.00MB
wp_wc_webhooks: Data: 0.00MB + Index: 0.00MB

### Post Type Counts ###

page: 6
post: 2
product: 4
shop_coupon: 1
shop_order: 10

### Security ###

Secure connection (HTTPS): ❌
					Your store is not using HTTPS. Learn more about HTTPS and SSL Certificates.
Hide errors from visitors: ✔

### Active Plugins (1) ###

WooCommerce: by Automattic – 3.5.2

### Settings ###

API Enabled: –
Force SSL: –
Currency: GBP (£)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)


### WC Pages ###

Shop base: #5 - /shop/
Cart: #6 - /cart/
Checkout: #7 - /checkout/
My account: #8 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Sixteen
Version: 1.6
Author URL: https://wordpress.org/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –
`
```

**Comment by rrennick** [comment]
Confirming this. On the front end the min spend check is being applied to the tax inclusive price. The order create/edit screen always uses the tax exclusive price so the product doesn't meet the min spend criteria.

**Comment by rrennick** [comment]
I tested this one with PR #22278 and was able to add the coupon in the dashboard.

**Comment by plxrisa** [comment]
@rrennick I checked your PR but I saw that is canceled. Also I don't think that the issue which I described is the same with #22239.

In my case, I have set (prices to be entered including tax). In the frontend it works fine when you try to redeem a coupon but in the backend (when you try to add a coupon by manually order, it doesn't work because the prices are displaying excluding tax).

So for example:
1. We create a coupon (ONLY for one usage) and share it via newsletter.
2. Someone call us to make an order via phone and he want to redeem his coupon.
3. We are not able to redeem it.
(If we make a discount manually without the coupon code, that means that he can redeem it in a future order, but he shouldn't).
